ansible/templates/topo/t1-lag.j2

```diff
@@ -0,0 +1,243 @@
+        <a:ManagementIPInterface>
+          <Name>HostIP</Name>
+          <AttachTo>eth0</AttachTo>
+          <a:Prefix xmlns:b="Microsoft.Search.Autopilot.Evolution">
+            <b:IPPrefix>{{ ansible_host }}/{{ mgmt_subnet_mask_length }}</b:IPPrefix>
+          </a:Prefix>
+          <a:PrefixStr>{{ ansible_host }}/{{ mgmt_subnet_mask_length }}</a:PrefixStr>
+        </a:ManagementIPInterface>
+      </ManagementIPInterfaces>
+      <MplsInterfaces/>
+      <MplsTeInterfaces/>
+      <RsvpInterfaces/>
+      <Hostname>{{ inventory_hostname }}</Hostname>
+      <PortChannelInterfaces>
+{% for index in range(8) %}
+{% set if_index = index*8 if ('Force10-S6000' in hwsku or 'MSN' in hwsku) else index*2+1 %}
+{% set if_next = index*8+4 if ('Force10-S6000' in hwsku or 'MSN' in hwsku) else index*2+2 %}
+        <PortChannel>
+          <ElementType>PortChannelInterface</ElementType>
+          <Name>PortChannel{{ if_index }}</Name>
+          <AttachTo>{{ iface_prefix }}{{ if_index }}{{ iface_postfix }};{{ iface_prefix }}{{ if_next }}{{ iface_postfix }}</AttachTo>
+           <SubInterface/> 
+         </PortChannel>
+{% endfor %}
+      </PortChannelInterfaces>
+      <VlanInterfaces/>
+      <IPInterfaces>
+{% for index in range(8) %}
+{% set if_index = index*4 if ('Force10-S6000' in hwsku or 'MSN' in hwsku) else index %}
+{% set po_index = index*8 if ('Force10-S6000' in hwsku or 'MSN' in hwsku) else index*2+1 %}
+        <IPInterface>
+          <ElementType>IPInterface</ElementType>
+          <Name i:Name="true"/>
+          <AttachTo>PortChannel{{ po_index }}</AttachTo>
+          <Prefix>10.0.0.{{ index*4 }}/31</Prefix>
+        </IPInterface>
+        <IPInterface>
+          <ElementType>IPInterface</ElementType>
+          <Name i:Name="true"/>
+          <AttachTo>PortChannel{{ po_index }}</AttachTo>
+          <Prefix>FC00::{{ ('%0x' % (index*8+1)) | upper }}/126</Prefix>
+        </IPInterface>
+{% endfor %}
+{% for index in range(16) %}
+{% set if_index = 64+index*4 if ('Force10-S6000' in hwsku or 'MSN' in hwsku) else index+17 %}
+        <IPInterface>
+          <ElementType>IPInterface</ElementType>
+          <Name i:Name="true"/>
+{% if 'Arista-7050-QX32' in hwsku and if_index > 24 %}
+          <AttachTo>{{ iface_prefix }}{{ if_index }}</AttachTo>
+{% else %}
+          <AttachTo>{{ iface_prefix }}{{ if_index }}{{ iface_postfix }}</AttachTo>
+{% endif %}
+          <Prefix>10.0.0.{{ 32+index*2 }}/31</Prefix>
+        </IPInterface>
+        <IPInterface>
+          <ElementType>IPInterface</ElementType>
+          <Name i:Name="true"/>
+{% if 'Arista-7050-QX32' in hwsku and if_index > 24 %}
+          <AttachTo>{{ iface_prefix }}{{ if_index }}</AttachTo>
+{% else %}
+          <AttachTo>{{ iface_prefix }}{{ if_index }}{{ iface_postfix }}</AttachTo>
+{% endif %}
+          <Prefix>FC00::{{ ('%0x' % (64+index*4 +1)) | upper }}/126</Prefix>
+        </IPInterface>
+{% endfor %}
```

Comment:
this is very difficult to maintain. it is better to use thi file to generate the interface name. [https://github.com/Azure/sonic-buildimage/blob/master/device/arista/x86_64-arista_7050_qx32/Arista-7050-QX32/port_config.ini](https://github.com/Azure/sonic-buildimage/blob/master/device/arista/x86_64-arista_7050_qx32/Arista-7050-QX32/port_config.ini)

In buildimage repo, it builds a debian package named sonic-device-data package. you can install the package in the sonic-mgmt docker and get the actual interface name based on port_config.ini for each hwsku.